charts/rclone-copy/templates/configmap.yaml

```diff
@@ -1,7 +1,12 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: rclone-config-{{ .Release.Name }}
+  name: rclone-config-{{ include "rclone-copy.fullname" . }}
```

Comment:
While you're at it, maybe rename to `{{ include "rclone-copy.fullname" . }}-rclone-config` or just simplify to `{{ include "rclone-copy.fullname" . }}` because it is a configmap: It is easier to find when coming from the deployment and resources belonging to a release are grouped

Reply:
Yeah also thought so. Was just a bit hesitant to break names here but helm should upgrade it properly